handlers/handlers.go

```diff
@@ -110,3 +115,72 @@ func GeographyRender(rend RenderClient) http.HandlerFunc {
 		return
 	}
 }
+
+//GeographyListpageRender ...
+func GeographyListpageRender(rend RenderClient) http.HandlerFunc {
+
+	return func(w http.ResponseWriter, req *http.Request) {
+		var page geographyListPage.Page
+
+		vars := mux.Vars(req)
+		areaTypeID := vars["areaTypeID"]
+
+		resp, err := http.Get(`https://api.dev.cmd.onsdigital.co.uk/v1/code-lists/` + areaTypeID + `/editions/2016/codes`)
+		fmt.Println("error err", err)
```

Comment:
we shouldn't use `fmt.Println` for logging - if this was temporary debugging we should remove it, otherwise we should use the `log` package from `go-ns`